corpus/convert.py

```diff
@@ -41,23 +41,18 @@ def run(self):
         orth_c.load(self.reference_bliss_corpus.get_path())
 
         orths = {}
-        for r in orth_c.all_recordings():
-            for i in range(len(r.segments)):
-                orth = r.segments[i].orth
-                tag = r.segments[i].fullname()
-                orths[tag] = orth
+        for s in orth_c.segments():
+            orth = s.orth
+            tag = s.fullname()
+            orths[tag] = name
 
         c = corpus.Corpus()
         c.load(self.bliss_corpus.get_path())
 
-        for r in c.all_recordings():
-            for i in range(len(r.segments)):
-                tag = r.segments[i].fullname()
-                assert tag in orths.keys(), (
-                    "Segment %s not found in reference corpus" % s
-                )
-                orth = orths[tag]
-                r.segments[i].orth = orth
+        for s in c.segments():
+            tag in s.fullname()
+            assert tag in orths.keys(), "Segment %s not found in reference corpus" % tag
+            s.orth = orths[tag]
 
         c.dump(self.out_corpus.get_path())
 
```

Comment:
```suggestion


class CorpusReplaceOrthFromReferenceCorpus(CorpusReplaceOrthFromReferenceCorpusJob):
    pass
    
```

Reply:
For my setup this does not yield new Jobs thus this should not break the hashes I think.

Reply:
yes this would work, but the missing "Job" in the name is only a cosmetic problem.

In my opinion, and I am open for discussion, keeping the cosmetic problem is a smaller evil than introducing additional classes.

some user who dislikes it could even do 
```
from i6_core.corpus.convert import CorpusReplaceOrthFromReferenceCorpus as CorpusReplaceOrthFromReferenceCorpusJob
```
and then happily continue their ways consistently.

Reply:
This works too, I am fine with both.